StyleCast/Views/UserInfoInputViewController.swift

```diff
@@ -0,0 +1,202 @@
+            // AppIcon
+            appIconView.topAnchor.constraint(
+                equalTo: view.safeAreaLayoutGuide.topAnchor, constant: 40
+            ),
+            appIconView.centerXAnchor.constraint(
+                equalTo: view.centerXAnchor
+            ),
+            appIconView.widthAnchor.constraint(equalToConstant: 144),
+            appIconView.heightAnchor.constraint(equalToConstant: 144),
+
+            // UsernameTextField
+            usernameTextField.topAnchor.constraint(
+                equalTo: appIconView.bottomAnchor, constant: 64
+            ),
+            usernameTextField.leadingAnchor.constraint(
+                equalTo: view.leadingAnchor, constant: 40
+            ),
+            usernameTextField.trailingAnchor.constraint(
+                equalTo: view.trailingAnchor, constant: -40
+            ),
+            usernameTextField.heightAnchor.constraint(equalToConstant: 48),
+
+            // AgeTextField
+            ageTextField.topAnchor.constraint(
+                equalTo: usernameTextField.bottomAnchor, constant: 16
+            ),
+            ageTextField.leadingAnchor.constraint(
+                equalTo: view.leadingAnchor, constant: 40
+            ),
+            ageTextField.trailingAnchor.constraint(
+                equalTo: view.trailingAnchor, constant: -40
+            ),
+            ageTextField.heightAnchor.constraint(equalToConstant: 48),
+
+            // GenderButtons
+            maleButton.topAnchor.constraint(
+                equalTo: ageTextField.bottomAnchor, constant: 30
+            ),
+            maleButton.leadingAnchor.constraint(
+                equalTo: view.leadingAnchor, constant: 40
+            ),
+            maleButton.heightAnchor.constraint(equalToConstant: 80),
+
+            femaleButton.topAnchor.constraint(
+                equalTo: ageTextField.bottomAnchor, constant: 32
+            ),
+            femaleButton.trailingAnchor.constraint(
+                equalTo: view.trailingAnchor, constant: -40
+            ),
+            femaleButton.heightAnchor.constraint(equalToConstant: 80),
+
+            maleButton.trailingAnchor.constraint(
+                equalTo: femaleButton.leadingAnchor, constant: -16
+            ),
+            maleButton.widthAnchor.constraint(equalTo: femaleButton.widthAnchor),
+
+            // RegisterButton
+            registerButton.bottomAnchor.constraint(
+                equalTo: view.safeAreaLayoutGuide.bottomAnchor, constant: -16
+            ),
+            registerButton.leadingAnchor.constraint(
+                equalTo: view.leadingAnchor, constant: 40
+            ),
+            registerButton.trailingAnchor.constraint(
+                equalTo: view.trailingAnchor, constant: -40
+            ),
+            registerButton.heightAnchor.constraint(equalToConstant: 48)
+        ])
+    }
+
+    private func setupActions() {
+        maleButton.addTarget(self, action: #selector(selectMale), for: .touchUpInside)
```

Comment:
[ここは変えても変えなくてもよし！]
`addAction`を使うと、objcのメソッドを作成しなくても済むので、全部swiftコードにしたいなら、おすすめ！
たとえば、こんな感じかな
```suggestion
        maleButton.addAction(.init { [weak self] _ in
            guard let self else { return }
            self.selectMale()
        }, for: .touchUpInside)
        ```
        func selectMale()
```

Reply:
知らなかった！ありがとう！